Yosemite/Yosemite/Stores/OrderStore.swift

```diff
@@ -737,18 +653,16 @@ private extension OrderStore {
     /// Updates (OR Inserts) the specified ReadOnly Order Entities *in a background thread*. onCompletion will be called
     /// on the main thread!
```

Comment:
"onCompletion will be called on the main thread!" is mentioned here but not on other places above (e.g: `upsertSearchResultsInBackground()` or `deleteStoredOrder()`) so I thought it might be better to make this consistent. 

imo the comment in `performAndSave()` already explains the usage so maybe it can be removed here.

Reply:
Comment updated in c62685f8dd9b505714eec84db6e3aa3e0053eeb9.